internal/devbox/devbox.go

```diff
@@ -1109,9 +1105,34 @@ func (d *Devbox) configEnvs(
 	ctx context.Context,
 	existingEnv map[string]string,
 ) (map[string]string, error) {
-	env, err := d.cfg.ComputedEnv(ctx, d.ProjectDir(), d.environment)
-	if err != nil {
-		return nil, err
+	env := map[string]string{}
+	if d.cfg.IsEnvsecEnabled() {
+		secrets, err := d.Secrets(ctx)
+		if err != nil {
+			return nil, err
+		}
+		envID, err := secrets.EnvID()
+		if err != nil {
+			return nil, err
+		}
+
+		cloudSecrets, err := secrets.List(ctx, envID)
+		if err != nil {
+			ux.Fwarning(
+				os.Stderr,
+				"Error reading secrets from jetpack cloud: %s\n\n",
+				err,
+			)
+		} else {
+			for _, secret := range cloudSecrets[envID] {
+				env[secret.Name] = secret.Value
+			}
+		}
+	} else if d.cfg.EnvFrom != "" {
+		return nil, usererr.New("unknown from_env value: %s", d.cfg.EnvFrom)
```

Comment:
`usererr.New("unknown from_env value: %s. Supported value is: envsec.", d.cfg.EnvFrom, )`